packages/voice/examples/recorder/package.json

```diff
@@ -14,15 +14,15 @@
 	"author": "Amish Shah <contact@shah.gg>",
 	"license": "MIT",
 	"dependencies": {
-		"@discordjs/opus": "^0.5.3",
 		"discord-api-types": "^0.30.0",
-		"discord.js": "^13.7.0",
+		"discord.js": "^13.8.0",
 		"libsodium-wrappers": "^0.7.9",
 		"node-crc": "^1.3.2",
-		"prism-media": "^2.0.0-alpha.0"
+		"opusscript": "^0.0.8",
+		"prism-media": "^1.3.2"
```

Comment:
This feels like a revert not bump 🤔

Reply:
Oops :) I misread the alpha version as an older one